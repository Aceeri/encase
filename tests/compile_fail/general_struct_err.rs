use encase::WgslType;

fn main() {}

#[derive(WgslType)]
struct Test;
