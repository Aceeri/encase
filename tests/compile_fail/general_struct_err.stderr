error: Only non empty structs with named fields are supported!
 --> tests/compile_fail/general_struct_err.rs:5:10
  |
5 | #[derive(WgslType)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `WgslType` (in Nightly builds, run with -Z macro-backtrace for more info)
